.github/CODEOWNERS

```diff
@@ -9,8 +26,11 @@
 algorithms/           @Qiskit/terra-core @manoelmarques @woodsp-ibm
 opflow/               @Qiskit/terra-core @manoelmarques @woodsp-ibm @ikkoham
 qiskit/utils/         @Qiskit/terra-core @manoelmarques @woodsp-ibm
```

Comment:
`qiskit.utils` is fairly general nowadays. I know I've put stuff in there that isn't Aqua-specific, so we might not need this rule any more.

Reply:
Well this was more about giving manoelmarques and woodsp-ibm ownership over the `QuantumInstance` and it's other support modules as they're the experts on that code. I can tweak this to try and restrict it beyond that I think I can list the files explicitly but I think it's probably ok like this because I trust them to approve PR only related to what they work on.

Reply:
Sure - I was thinking about making it easier for them to limit their notifications than to necessarily remove their permissions.